srtcore/core.cpp

```diff
@@ -11796,7 +11795,7 @@ static bool getOptDefault(SRT_SOCKOPT optname, void* pw_optval, int& w_optlen)
 
     case SRTO_SENDER: RD(false);
     case SRTO_TSBPDMODE: RD(false);
-    case SRTO_TSBPDDELAY:
+    case SRTO_LATENCY:
```

Comment:
Renaming this breaks building vlc against srt 1.4.2.

A more specific commit message in regards to this change and why it was changed would have been nice and helpful